src/Api.js

```diff
@@ -0,0 +1,21 @@
+import App from './App';
+
+class Api {
+  constructor(endpoint) {
+    this.url = 'https://rancid-tomatillos.herokuapp.com/api/v2' + endpoint;
+    this.movieData = {};
+    this.getMovies = this.getMovies.bind(this);
+  }
+
+  getMovies() {
```

Comment:
fetch looks great. This is a great start to getting to the rest of the display elements. Great work